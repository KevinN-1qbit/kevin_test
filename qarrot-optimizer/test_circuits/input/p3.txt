Rotate 2: IZIX
Rotate -2: IZII
Rotate -2: IIIX
Rotate 2: ZXII
Rotate -2: ZIII
Rotate -2: IXII
Rotate 2: XIIZ
Rotate -2: IIIZ
Rotate -2: XIII
Rotate 2: IZII
Rotate 2: IXII
Rotate 2: IZII
Rotate 2: IIZI
Rotate 2: IIXI
Rotate 2: IIZI
Rotate 1: ZIII
Rotate 2: IIZI
Rotate 2: IIXI
Rotate 2: IIZI
Rotate 1: ZIII
