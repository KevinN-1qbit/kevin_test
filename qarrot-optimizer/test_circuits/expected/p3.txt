Measure +: ZZIZ
Measure +: IXIX
Measure +: IIZI
Measure +: IZIZ
